branch_delete_except.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+for branch in `git branch`; do
```

Comment:
It looks like this file is not related to this patch, please remove

Reply:
Oh, my mistake.